server/data/helpers/studentsHelper.js

```diff
@@ -0,0 +1,60 @@
+const db = require('../../config/dbConfig');
+
+module.exports = {
+	getAll: async () => {
+		const allStudents = await db('students').select(
+			'id',
+			'firstname',
+			'lastname',
+			'email'
+		);
+		return allStudents;
+	},
+
+	getStudent: async (id) => {
+		const student = await db('students')
+			.select('id', 'firstname', 'lastname', 'email')
```

Comment:
Same as before, don't think the select is necessary since we're selecting all fields, but not a big deal either way. Maybe it's better practice since we could possibly add fields later. :slightly_smiling_face:

Reply:
Good catch,  I will have to go through and look for these